content/2018-10-30_paty-sraz-enum-dataclasses.md

```diff
@@ -31,17 +30,17 @@ V další části jsme se naučily něco nového, využití knihoven **enum** a
 `python -m pip install dataclasses`
 `python -m pip install enum`
  
-Tyto knihovny nám umožní jednodušeji psát výčtové typy a datové třídy. Kdo nezná třídy, zopakuje si (https://naucse.python.cz/course/pyladies/sessions/class/)
+Tyto knihovny nám umožní jednodušeji psát výčtové typy a datové třídy. Kdo nezná třídy, zopakuje si ze začátečnického kurzu [Třídy](https://naucse.python.cz/course/pyladies/sessions/class/)
  
 Poté jsme v editoru vytvořily nový soubor – demo.py kde máme třídu Direction, která dědí z výčtového typu enum.Enum. Definovaly jsme si funkci, která vyhodnotí směr (nejdřív se sečtou hodnoty a zbytek po dělení 4 => získáme směr).
 
 
 ![demo](./images/demo.jpg)
 
 
-Dále následovalo vysvětlování modulu **dataclasses**. Tento modul poskytuje dekorátor a funkce pro automatické přidávání generovaných speciálních metod, jako jsou __init __ () a __repr __ () do uživatelsky definovaných tříd. 
+Dále následovalo vysvětlování modulu **dataclasses**. Tento modul poskytuje dekorátor a funkce pro automatické přidávání generovaných speciálních metod, jako jsou `__init __ ()` a `__repr __ ()` do uživatelsky definovaných tříd. 
```

Comment:
V tom `__init __ ()` a `__repr __ ()` je nějak moc mezer, neměly by tam být žádné

Reply:
Ja vim netuším jakto že tam jsou když jsem to kopírovala :-D snad opraveno.